src/storage/store/node_column.cpp

```diff
@@ -437,6 +487,8 @@ std::unique_ptr<NodeColumn> NodeColumnFactory::createNodeColumn(const LogicalTyp
     BMFileHandle* metadataFH, BufferManager* bufferManager, WAL* wal) {
     switch (dataType.getLogicalTypeID()) {
     case LogicalTypeID::BOOL:
+        return std::make_unique<BoolNodeColumn>(
```

Comment:
Wrap this line around `{}`. We are following the code style of always wrapping case statements inside `{}`.